.circleci/config.yml

```diff
@@ -2,12 +2,12 @@
 #  Each post contains a full image manifest, including iOS runtimes, devices, CocoaPods version, etc.
 #  NOTE: When updaing Xcode check the manifest for compatible chruby versions.
 anchors:
-  - &latest-xcode    "13.4.1"
-  - &latest-ios      "15.5"
-  - &min-ios         "14.5"
-  - &requres-string  "test << matrix.lib >> iOS 14.5"  # Keep this version up to date
+  - &latest-xcode    "14.1.0"
```

Comment:
It looks like the CircleCI 14.2.0+ images have some breaking changes so I updated to `14.1.0` to unblock the iOS 15 bump and then can play around with the config more for updating to `14.3.1`

Reply:
👍